diff --git a/chrome/test/BUILD.gn b/chrome/test/BUILD.gn
index 0dc0d9aea8d99b62b49ddd61ccd8d63510f80a72..4c3ce78420506f6b67fd5f1dcd97e2dc43f56279 100644
--- a/chrome/test/BUILD.gn
+++ b/chrome/test/BUILD.gn
@@ -760,6 +760,7 @@ if (!is_android) {
       "//v8",
     ]
 
+    deps += [ "//brave/test:brave_browser_tests_deps", ]
     # Runtime dependencies
     data_deps = [
       "//chrome:browser_tests_pak",
