diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
index f9e4b1176bb34c5cf137ee1dd2bfbc24f1426a13..c5563ea1203e2920344a653cb822a527600a45e4 100644
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -2255,6 +2255,7 @@ jumbo_static_library("browser") {
     "//ui/strings",
     "//ui/surface",
     "//ui/web_dialogs",
+    "//brave/browser:browser_process",
   ]
 
   if (is_win) {
