diff --git a/chrome/browser/resources/settings/BUILD.gn b/chrome/browser/resources/settings/BUILD.gn
index 584a8a83e4aa95a737c2662c3c2d14762d90b0aa..1ad3a9c2f49830fbde96d99027722bd339dc23f8 100644
--- a/chrome/browser/resources/settings/BUILD.gn
+++ b/chrome/browser/resources/settings/BUILD.gn
@@ -33,6 +33,7 @@ if (optimize_webui) {
     excludes = [ "chrome://resources/mojo/chromeos/services/network_config/public/mojom/cros_network_config.mojom.html" ]
 
     deps = [ ":unpak" ]
+    deps += [ "//brave/browser/resources/settings:unpak" ]
   }
 
   unpak("unpak") {
