diff --git a/chrome/android/java/src/org/chromium/chrome/browser/toolbar/bottom/BottomToolbarCoordinator.java b/chrome/android/java/src/org/chromium/chrome/browser/toolbar/bottom/BottomToolbarCoordinator.java
index 19475da4bf833ec8f517497cd413499769ce32ec..3e5e96cdf0d1db5a868fdfc027158b429d63bb1e 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/toolbar/bottom/BottomToolbarCoordinator.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/toolbar/bottom/BottomToolbarCoordinator.java
@@ -164,4 +164,5 @@ class BottomToolbarCoordinator {
         }
         mThemeColorProvider.destroy();
     }
+    public void updateBookmarkButton(boolean isBookmarked, boolean editingAllowed) {if (mBrowsingModeCoordinator != null) {mBrowsingModeCoordinator.updateBookmarkButton(isBookmarked, editingAllowed);}}
 }
