diff --git a/chrome/browser/permissions/permission_uma_util.cc b/chrome/browser/permissions/permission_uma_util.cc
index e46e06f7cdce54937daead401b16f64f7230e26d..1d9f4fb5ec82df5881988d82a19bcb820805b55c 100644
--- a/chrome/browser/permissions/permission_uma_util.cc
+++ b/chrome/browser/permissions/permission_uma_util.cc
@@ -68,7 +68,7 @@ namespace {
 
 const int kPriorCountCap = 10;
 
-std::string GetPermissionRequestString(
+std::string GetPermissionRequestString_ChromiumImpl(
     permissions::PermissionRequestType type) {
   switch (type) {
     case permissions::PermissionRequestType::MULTIPLE:
@@ -572,6 +572,7 @@ void PermissionUmaUtil::RecordPermissionAction(
       base::UmaHistogramEnumeration("Permissions.Action.AR", action,
                                     permissions::PermissionAction::NUM);
       break;
+    BRAVE_PERMISSIONUMAUTIL_RECORDPERMISSIONACTION
     // The user is not prompted for these permissions, thus there is no
     // permission action recorded for them.
     default:
