diff --git a/chrome/browser/ui/webui/settings/settings_import_data_handler.cc b/chrome/browser/ui/webui/settings/settings_import_data_handler.cc
index b2e78bd722b68c0eca2962ee1f399d1edc8114f5..8c3bee44ed253354668a447034c2863cca00f59e 100644
--- a/chrome/browser/ui/webui/settings/settings_import_data_handler.cc
+++ b/chrome/browser/ui/webui/settings/settings_import_data_handler.cc
@@ -126,6 +126,7 @@ void ImportDataHandler::ImportData(const base::ListValue* args) {
     selected_items |= importer::PASSWORDS;
   if (prefs->GetBoolean(prefs::kImportDialogSearchEngine))
     selected_items |= importer::SEARCH_ENGINES;
+  ADD_BRAVE_SPECIFIC_IMPORTER_ITEMS
 
   const importer::SourceProfile& source_profile =
       importer_list_->GetSourceProfileAt(browser_index);
@@ -179,6 +180,7 @@ void ImportDataHandler::SendBrowserProfileData(const std::string& callback_id) {
     browser_profile->SetBoolean(
         "autofillFormData",
         (browser_services & importer::AUTOFILL_FORM_DATA) != 0);
+    SET_BRAVE_SPECIFIC_IMPORTER_DATA
 
     browser_profiles.Append(std::move(browser_profile));
   }
