diff --git a/chrome/browser/ui/cocoa/main_menu_builder.mm b/chrome/browser/ui/cocoa/main_menu_builder.mm
index 9a59c30aaf0be88ab641051c2fc95f8b8be7acf2..c7aa06239193451cf90ea8db3894701d3ac8f999 100644
--- a/chrome/browser/ui/cocoa/main_menu_builder.mm
+++ b/chrome/browser/ui/cocoa/main_menu_builder.mm
@@ -99,6 +99,7 @@ base::scoped_nsobject<NSMenuItem> BuildFileMenu(
                 Item(IDS_NEW_INCOGNITO_WINDOW_MAC)
                     .command_id(IDC_NEW_INCOGNITO_WINDOW)
                     .remove_if(is_pwa),
+                BRAVE_BUILD_FILE_MENU
                 Item(IDS_REOPEN_CLOSED_TABS_MAC)
                     .command_id(IDC_RESTORE_TAB)
                     .remove_if(is_pwa),
@@ -446,6 +447,7 @@ base::scoped_nsobject<NSMenuItem> BuildHelpMenu(
 #if BUILDFLAG(GOOGLE_CHROME_BRANDING)
             Item(IDS_FEEDBACK_MAC).command_id(IDC_FEEDBACK),
 #endif
+                BRAVE_BUILD_HELP_MENU
                 Item(IDS_HELP_MAC)
                     .string_format_1(product_name)
                     .command_id(IDC_HELP_PAGE_VIA_MENU),
