diff --git a/chrome/android/chrome_public_apk_tmpl.gni b/chrome/android/chrome_public_apk_tmpl.gni
index a821df0dfa9eec5daf4841354d9a01176e23264f..c3ddbdc9795902e6c4dff3d4bc675dcd3f1e84b0 100644
--- a/chrome/android/chrome_public_apk_tmpl.gni
+++ b/chrome/android/chrome_public_apk_tmpl.gni
@@ -253,6 +253,7 @@ template("chrome_public_common_apk_or_module_tmpl") {
       if (!defined(proguard_configs)) {
         proguard_configs = []
       }
+      proguard_configs += ["//brave/android/java/proguard.flags",]
       proguard_configs += [
         "//chrome/android/java/proguard.flags",
         "//base/android/proguard/chromium_apk.flags",
