diff --git a/components/sync/driver/glue/sync_engine_impl.h b/components/sync/driver/glue/sync_engine_impl.h
index 098bc6807a9d0f731486f787f8d33485ecb9d11b..7f8fa86eb7ee753a5ac9da0af313f8be8381fc24 100644
--- a/components/sync/driver/glue/sync_engine_impl.h
+++ b/components/sync/driver/glue/sync_engine_impl.h
@@ -221,6 +221,7 @@ class SyncEngineImpl : public SyncEngine, public InvalidationHandler {
 
   base::WeakPtrFactory<SyncEngineImpl> weak_ptr_factory_{this};
 
+  BRAVE_SYNC_ENGINE_IMPL_H
   DISALLOW_COPY_AND_ASSIGN(SyncEngineImpl);
 };
 
