diff --git a/content/public/test/browser_test_utils.cc b/content/public/test/browser_test_utils.cc
index 7cdfedcb656c1bd8560068403272539cca2be972..67631a929a968d5cb9e3cae6781631c3beeea351 100644
--- a/content/public/test/browser_test_utils.cc
+++ b/content/public/test/browser_test_utils.cc
@@ -1909,6 +1909,7 @@ bool SetCookie(BrowserContext* browser_context,
       ->GetNetworkContext()
       ->GetCookieManager(cookie_manager.BindNewPipeAndPassReceiver());
   std::unique_ptr<net::CanonicalCookie> cc(net::CanonicalCookie::Create(
+      BRAVE_SETCOOKIE
       url, value, base::Time::Now(), base::nullopt /* server_time */));
   DCHECK(cc.get());
 
